src/test/java/io/vertx/tests/contract/OpenAPIContractTest.java

```diff
@@ -216,4 +216,17 @@ public void testAdditionalSchemaFiles(Vertx vertx, VertxTestContext testContext)
       })));
   }
 
+  @Test
+  @Timeout(value = 2, timeUnit = TimeUnit.SECONDS)
+  public void testVendorSpecificJson(Vertx vertx, VertxTestContext testContext) {
```

Comment:
increase timeout for CI, 10 seconds is reasonnable

Reply:
I was just following the pattern used across the entire library which is just about 2 seconds. Increasing here would mean also increasing everywhere for consistency. Maybe in a separate PR imho.